pom.xml

```diff
@@ -78,67 +51,39 @@
   </distributionManagement>
 
   <properties>
-    <mojo.java.target>1.7</mojo.java.target>
-    <mavenVersion>3.5.2</mavenVersion>
+    <mojo.java.target>1.8</mojo.java.target>
+    <mavenVersion>3.8.2</mavenVersion>
```

Comment:
Updating to 3.8.3 or greater versions breaks the Mojo tests due to the dependencies update in plexus, see z https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12316922&version=12350518 
More investigation is required to fix the tests